CODEOWNERS

```diff
@@ -17,6 +17,7 @@
 /.github/				  @nashif
 /.github/workflows/                       @galak @nashif
 /.buildkite/				  @galak
+/MAINTAINERS.yml                          @ioannisg @MaureenHelm
 /arch/arc/                                @vonhust @ruuddw
```

Comment:
@vonhust is leaving Synopsys, so pls mention @abrodkin instead.

Reply:
This is done now @abrodkin